ai_transform/workflow/abstract_workflow.py

```diff
@@ -121,6 +121,45 @@ def run(self):
         except WorkflowFailedError as e:
             logger.exception(e)
 
+        else:
+            n_processed_pricing = self._calculate_pricing()
+            if n_processed_pricing is not None:
+                self.update_workflow_pricing(n_processed_pricing)
+
+    def _calculate_pricing(self):
+        n_processed_pricing = 0
+        is_automatic = False
+
+        for operator in self.operators:
+            is_automatic &= not operator.is_operator_based_pricing
```

Comment:
```
automatic = True
if operator.is_operator_based_pricing: 
    is_automatic = False
```

Reply:
In programming, the first condition is widely considered much harder to follow than the second.
```
if not is_XYZ:
    do_something() 
-> 
if is_XYZ:
    do_something()
```